python/Ganga/GPIDev/Schema/Schema.py

```diff
@@ -402,7 +402,12 @@ def __len__(self):
     # item.isA(SimpleItem)
     def isA(self, _what):
 
-        from Ganga.GPIDev.Base.Proxy import stripProxy
+        try:
+            from Ganga.GPIDev.Base.Proxy import stripProxy
+        except ImportError:
+            def dummyfunc(arg):
+                return arg
+            stripProxy = dummyfunc
```

Comment:
Under what situation does this `ImportError` occur? I don't like complicated logic like this in such a central bit of code.